src/bootstrap/investment-sheet-boostrapper.js

```diff
@@ -6,39 +6,56 @@ class InvestmentSheetBootstrapper extends BaseSheetBootstrapper {
 
   createSheet() {
     const numRows = 200;
+    const letterColDate = colToLetter(TransactionsSheet.columnDate);
+    const letterColSource = colToLetter(TransactionsSheet.columnSource);
+    const letterColIncome = colToLetter(TransactionsSheet.columnIsIncome);
+    const letterColInvestment = colToLetter(TransactionsSheet.columnIsInvestment);
     this.investmentsTable = new DataTable(
       this.sheet,
       2,
       2,
       'Investment',
-      ['Date', 'Amount', 'Description', 'Category', 'Source'],
+      ['Date', 'Amount', 'Description', 'Category', 'Classificator', 'Source'],
```

Comment:
This sheet still allows editing (and bugs out when editing anything). Let's get rid of it (make it akin to the new months sheet).

Reply:
And it also has the "Refresh" button that we can remove

Reply:
done